Automattic/jetpack
# Add/additional menu items to domain only sidebar

This set of changes adds the necessary menu items that makes sense for domain-only sites, with the conditional addition of  a menu item `Manage Email`  when a site contains an active email subscription.

On simple sites, a version of the menu looks like below:

<img width="1163" alt="Screenshot 2022-09-08 at 6 07 07 AM" src="https://user-images.githubusercontent.com/277661/189040908-19f93b8d-37d2-4df7-aab8-01ed73aadfaa.png">

At the moment, domain-only features cannot be established on Atomic sites, hence this PR only ensures that shared logic between the two codebases are in sync as required.




#### Changes proposed in this Pull Request:

* Adds menu items:
    * Manage Domain
    * Manage Email ( conditionally present when an email subscription is present )
    * Manage Purchases 
    * Inbox

#### Other information:


#### Jetpack product discussion

pbLl1t-1f6-p2#comment-1252


#### Does this pull request change what data or activity we track or use?



#### Testing instructions:







- Ensure that you have the [Jetpack Beta](https://jetpack.com/download-jetpack-beta/) plugin uploaded and installed on your Atomic test site.
- Use Jetpack Beta to switch to this branch: `add/additional-menu-items-to-domain-only-sidebar`
- Refresh the admin page you're on, and verify that the menu hasn't changed in any way. 
- Specifically you would want to ensure that the following menu items do not show up:
    - `Manage Domain`
    - `Manage Email`
    - `Manage Purchases`
